docs/report/report.tex

```diff
@@ -385,14 +385,60 @@ \subsubsection{Retrospektive}
 \colorbox{red}{TODO:}Was haben wir dabei gelernt? Neue Erkentnisse? Neue Sichtweisen?
 Was lief gut, neu gelernt, was lief nicht so gut, was verbessern?
 
-\section{Erkenntnisse}
-
 \section{Code Walkthrough}
 \colorbox{red}{TODO:}vielleicht interessante Code Snippets?
 
 \section{Testing}
 \colorbox{red}{TODO:}wie haben wir getestet, haben wir getestet?
 
+\section{Fazit}
+\subsection{Probleme, Lösungen, Erkenntnisse}
+\colorbox{red}{TODO: Finde TITEL:} Fazit oder nur Erkenntnisse oder oder?
+\colorbox{red}{Besseres Intro zum Kapitel?}
+Das Projekt Informatik gab einen Einblick in die Arbeitswelt, das Team musste sich klassischen Alltagsproblemen stellen, konnte Lösungswege finden und wichtige Erkenntnisse für die Zukunft daraus ziehen.
+Auch Coding-Entwurfsmuster und -Konventionen wurden neu erlernt und praktisch angewandt.
+
+\subsubsection{Einarbeitung in neue Technologien}
+Zu Beginn des Projektes bestand eine größere Differenz, inwieweit die einzelnen Teammitglieder mit den jeweiligen Technologien vertraut waren.
+Sich mit der Projektstruktur vertraut zu machen war eine größere Hürde.
+Durch die vielen automatisch generierten Dateien und Ordner wirkte die Komplexität und somit Einstiegshürde groß, und Teammitglieder brauchten Ihre Zeit, um effektiv mitarbeiten zu können.
+Über die Entwicklungszeit wurden aber die neuen Technologien vertieft und verinnerlicht.
+Das Team lernte auch wieso Frameworks und Librarys wie Vue, Tailwind und Feathers sinvoll und hilfreich sein können.
+
+\subsubsection{Versionskontrolle und Deployment Cycle}
+Im Studium wurde den Studierenden der Vorteil von Versionskontrollsystemen wie Git vermittelt, doch fand sich im Studium selten ein praktisches Anwendungszenario, dass Git über die Hauptfunktion der Versionierung hinaus sinnvoll genutzt hat.
+Durch dieses Projekt fand sich das erste mal im Studium eine sinnvolle Anwendung, wie in größeren Projekten oder in der Industrie auch mit Pull Requests und Code Reviews zu arbeiten.
+Dadurch traten mehrere Effekte auf:
+Wenn mehrere Teammitglieder über den Code anderer schauten, fielen umso wahrscheinlicher Fehler auf und eine insgesamt bessere Codequalität wurde sichergestellt.
+Code Reviews hatten außerdem den Effekt, dass Teammitglieder neue Funktionen und Änderungen kennenlernen, verstehen und gegebenenfalls auch direkt Fragen dazu stellen konnten.
+Sich an diesen Deployment Cycle zu gewöhnen brauchte eine gewisse Zeit, aber fand von Sprint zu Sprint immer mehr Wertschätzung im Team.
```

Comment:
```suggestion
Diese Methoden führten zu folgenden maßgebliche Verbesserungen:
Durch die erforderten zwei Code Reviews pro Pull Request wurde Fehler im Code häufiger entdeckt und eine höhere Codequalität wurde sichergestellt. Als positiver Nebeneffekt konnten die Reviewer außerdem während des Prozesses neue Funktionen und Codemuster erlernen, zu denen sie bei Bedarf (per Kommentar) Fragen stellen konnten.


Wenn mehrere Teammitglieder über den Code anderer schauten, fielen umso wahrscheinlicher Fehler auf und eine insgesamt bessere Codequalität wurde sichergestellt.
Code Reviews hatten außerdem den Effekt, dass Teammitglieder neue Funktionen und Änderungen kennenlernen, verstehen und gegebenenfalls auch direkt Fragen dazu stellen konnten.
Sich an diesen Deployment Cycle zu gewöhnen brauchte eine gewisse Zeit, aber fand von Sprint zu Sprint immer mehr Wertschätzung im Team.
```

Reply:
Danke, fand vorher kein gutes deutsches wort zu "coding schemes"

Reply:
umgesetzt in https://github.com/project-leek/project-leek/pull/419/commits/c5dae29b261f99d39aac66c889e6bb96205a2f1a